### ISSUE TITLE
I Use  Record 5.0.1 ver Crash

### ISSUE THREAD

**Issue by DreamHeader** [llfbandit/record#230]
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: '*** -[AVAssetWriter finishWritingWithCompletionHandler:] Cannot call method when status is 1’

![image](https://github.com/llfbandit/record/assets/35476529/2595f81c-4734-4719-a691-bd27d798bc17)

**Comment by llfbandit** [comment]
iOS, macOS? Steps to reproduce?
There's a template to get minimal info...

**Comment by llfbandit** [comment]
Should be fixed in `record_darwin: ^1.0.1-beta.1`. You will need to add it manually to test it.

**Comment by docaohuynh** [comment]
I have got same error on IOS
```
Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: '\*\*\* -[AVAssetWriter finishWritingWithCompletionHandler:] Cannot call method when status is 1'
```

**Comment by llfbandit** [comment]
Please re-test this with `record_darwin: ^1.0.1-beta.1`.
`AVAssetWriter` is not used anymore in this version.

**Comment by docaohuynh** [comment]
tks @llfbandit after update to `record_darwin: ^1.0.1-beta.1. ` seem like it work perfectly

**Comment by DreamHeader** [comment]
> iOS, macOS? Steps to reproduce? There's a template to get minimal info...

I use iOS。

**Comment by DreamHeader** [comment]
> tks @llfbandit after update to `record_darwin: ^1.0.1-beta.1. ` seem like it work perfectly

hello , I don't find some lib about  `record_darwin". how do you find ? what url? website?

**Comment by docaohuynh** [comment]
@DreamHeader you can clone this repository and change `pubspec.yaml` in local

**Comment by DreamHeader** [comment]
> Or just add it to your project. The package is on pub.dev but unlisted.

now I. add record_darwin: ^1.0.1-beta.1.  Record need delete??????

**Comment by llfbandit** [comment]
Released in record_darwin 1.0.1
